Dockerfile

```diff
@@ -1,27 +1,44 @@
-ARG agent_version=4.6-1
-FROM jenkins/inbound-agent:${agent_version}-alpine as install
+ARG agent_version=4.10-3
+
+FROM ubuntu:20.04 as sessionmanagerplugin
+
+RUN apt-get update \
+    && apt-get install -y curl \
+    && curl -Lo "session-manager-plugin.deb" "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/ubuntu_64bit/session-manager-plugin.deb" \
+    && dpkg -i "session-manager-plugin.deb"
+
+
+FROM jenkins/inbound-agent:${agent_version}-alpine-jdk11 as install
 
 # Elevate
 USER root
 #  need to install both npm and nodejs
-ARG nodejs_version=16.13.2-r1
-ARG npm_version=12.22.10-r0
+ARG nodejs_version=16.19.1-r0
+ARG npm_version=8.1.3-r0
```

Comment:
Why the npm change?

Reply:
npm change was just matching master.
nodejs change is the only 16 version thats still in the alpinelinux repo